src/storage.rs

```diff
@@ -92,19 +93,39 @@ pub struct Eip7702Code {
     version: u8,
 }
 
+/// EVM Object Format (EOF) code.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct EofCode {
+    /// EOF header.
+    pub header: EofHeader,
```

Comment:
I haven't looked deeply into EOF, but looking at `Eof::new` and `EofBody::into_eof`, the header can be inferred from the body? If so we can make this `EofCode` struct equivalent to `revm::EofBody`.

Reply:
Yes, `EofHeader` can be conducted from `EofBody`; but I am not very sure if we will use fields of EofHeader later.

Reply:
Not directly. For now, we're just converting these types to `revm` types at runtime.